### DISCUSSION TITLE
Units and location of posture data

### DISCUSSION THREAD

**Discussion by walsmanjason** [mooch443/trex#218] [Questions]
I am able to output posture data and access the offset, outline_points, etc. However, I am not quire clear on the units of these data. offset is the top left corner of the blob in pixels given as XY coordinates, I understand from the documentation. But what are the units of outline_points? Are those XY coordinate pairs also in units of pixels? And are they given relative to the offset?

**Comment by walsmanjason** [comment]
Just commenting to ping my question again.

**Comment by mooch443** [comment]
Hey!
You can have a look here:
https://github.com/mooch443/trex/blob/4839c64f0bdb87cacbe4d12edf71eadd6d7f3069/docs/scripts/plot_posture_output.py#L80
The x/y coordinates of the outline_points is in the same coordinate system as the offsets (which are added to get the absolute/global position of each outline point). So its pixel coordinates relative to blob coordinates.
